sources/api/types.go

```diff
@@ -57,19 +57,27 @@ func (a *AggregateData) Merge(b *AggregateData) {
 	a.Events = append(a.Events, b.Events...)
 }
 
+type ContainerSpec struct {
+	cadvisor.ContainerSpec
+	CpuRequest    int64
```

Comment:
Having these 2 grouped in a struct makes sense but probably the better option is what @vishh described in https://github.com/kubernetes/heapster/pull/539#discussion_r39319547.

As @mwielgus noticed cpu limit is already computed by cadvisor. Let's remove cadvisor dep in a new PR (probably after 1.1). AFAIK there is no requests computed by cadvisor so we need to workaround it by obtaining the data from Kubernetes.